geneLevel.R

```diff
@@ -41,91 +56,111 @@ IMPACT468_targets = IMPACT468_targets[V5 %like% 'target']
 setnames(IMPACT468_targets, c("chr", "start", "end", "strand", "name"))
```

Comment:
At some point, we should make a local copy of this in the repo, and update gene names to match OncoKB. For now, I can confirm that all TSG gene names match OncoKB. Among non-TSGs, only 3 have been renamed: `SETD8->KMT5A`, `MYCL1->MYCL`, and `PAK7->PAK5`.

Reply:
Sounds good, feel free to make a PR

Reply:
I can do this real quick today for the 3 IMPACT assays. I have a script at https://github.com/mskcc/cmo-scripts to update gene names. This will also remove our dependency on files stored at `/ifs/depot`.

Reply:
Done. Created subfolder `data` for these target files, and moved other datafiles into it too. Scripts that use those datafiles have also been updated as needed. Note that 9 gene names will remain outdated to match what OncoKB uses (FAM175A, FAM46C, FAM58A, MRE11A, PARK2, RFWD2, TCEB1, WHSC1, WHSC1L1).